hack/vault-init.sh

```diff
@@ -172,9 +173,9 @@ function approleAuth() {
   cat << EOF
 
 secret yaml with Vault credentials prepared
-make sure your kubectl context targets cluster with SPI deployment and create the secret using (check spi namespace):
+make sure your kubectl --kubeconfig=${VAULT_KUBE_CONFIG} context targets cluster with SPI deployment and create the secret using:
```

Comment:
I would say this is wrong. you want to apply to SPI deployment, so it's not vault kubeconfig.